io.mpv.Mpv.json

```
@@ -0,0 +1,155 @@
+{
+  "app-id": "io.mpv.Mpv",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mpv",
+  "rename-desktop-file": "mpv.desktop",
+  "rename-icon":"mpv",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=all",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--filesystem=host",
```

Does MPV ever need write access? If not,
```suggestion
    "--filesystem=host:ro",
```

---

It need write access for plugins.

---

Plugins are stored in MPV config dir, which is mounted read-writable even with `host:ro`.

---

If you will test all subtitle addons and write the report, we can continue the discuss.

---

Can you point to a specific plugin that writes to a directory outside xdg-config/xdg-data?

---

Guess all subtitle plugins allows to download subtitles to multimedia folder.

---

Isn't it too specific use-case to expose whole filesystem for _all_ users? Users who actually need write access to host, may add it via overrides.

---

It'd be appreciated if you could stop resolving conversations that are still in progress.

---

Screenshot needs write access, though I guess only to `xdg-pictures`.